designer/server/src/test/scala/pl/touk/nussknacker/ui/api/NuDesignerApiAvailableToExposeYamlSpec.scala

```diff
@@ -0,0 +1,17 @@
+package pl.touk.nussknacker.ui.api
+
+import better.files.Dsl._
+import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.matchers.should.Matchers
+import pl.touk.nussknacker.ui.services.NuDesignerApiAvailableToExpose
+
+// if the test fails it probably means that you should regenerate the Nu Designer OpenAPI document
+// you can do it but running manually the object `GenerateDesignerOpenApiYamlFile` with main method or
+// using SBT's task: `sbt generateDesignerOpenApi`
+class NuDesignerApiAvailableToExposeYamlSpec extends AnyFunSuite with Matchers {
+
+  test("Nu Designer OpenAPI document with all available to expose endpoints has to be up to date") {
+    val currentNuDesignerOpenApiYamlContent = (pwd / "docs" / "api" / "internal" / "nu-designer-openapi.yaml").contentAsString
```

Comment:
Please don't user working directory. It can behave differently depending on many external things. Why not to use resources instead?

Reply:
ref: https://github.com/TouK/nussknacker/pull/4709#discussion_r1342640241